Get rid of console.log statements

Current status:

```sh
$ npm run lint

...

> firefox-send@0.2.0 lint:js /home/ubuntu/send
> eslint .


/home/ubuntu/send/frontend/src/download.js
  105:9  warning  Unexpected console statement  no-console
  156:9  warning  Unexpected console statement  no-console

/home/ubuntu/send/frontend/src/fileSender.js
  29:11  warning  Unexpected console statement  no-console
  31:11  warning  Unexpected console statement  no-console
  57:11  warning  Unexpected console statement  no-console

/home/ubuntu/send/frontend/src/upload.js
  212:9   warning  Unexpected console statement  no-console
  224:9   warning  Unexpected console statement  no-console
  300:11  warning  Unexpected console statement  no-console

✖ 8 problems (0 errors, 8 warnings)
```

We should either remove all unnecessary `console.log()` statements (and/or ESLint ignore any valid ones), or possibly just remove the ESLint rule to avoid console vomit in Circle-CI and when linting locally.

--- a/frontend/src/download.js
+++ b/frontend/src/download.js
@@ -90,7 +90,6 @@ $(document).ready(function() {
           $('.progress-text').text(decryptingFile);
         });
       } else {
-        console.log('Done decrypting');
         downloadEnd = Date.now();
       }
     });

--- a/frontend/src/fileSender.js
+++ b/frontend/src/fileSender.js
@@ -46,10 +46,6 @@ class FileSender extends EventEmitter {
           },
           true,
           ['encrypt', 'decrypt']
-        )
-        .catch(err =>
-          // eslint-disable-next-line no-console
-          console.error('There was an error generating a crypto key')
         ),
       new Promise((resolve, reject) => {
         const reader = new FileReader();

--- a/frontend/src/fileSender.js
+++ b/frontend/src/fileSender.js
@@ -24,12 +24,6 @@ class FileSender extends EventEmitter {
         if (xhr.readyState === XMLHttpRequest.DONE) {
           resolve();
         }
-
-        if (xhr.status === 200) {
-          console.log('The file was successfully deleted.');
-        } else {
-          console.log('The file has expired, or has already been deleted.');
-        }
       };
 
       xhr.send(JSON.stringify({ delete_token: token }));
@@ -54,7 +48,8 @@ class FileSender extends EventEmitter {
           ['encrypt', 'decrypt']
         )
         .catch(err =>
-          console.log('There was an error generating a crypto key')
+          // eslint-disable-next-line no-console
+          console.error('There was an error generating a crypto key')
         ),
       new Promise((resolve, reject) => {
         const reader = new FileReader();

--- a/frontend/src/upload.js
+++ b/frontend/src/upload.js
@@ -206,8 +206,6 @@ $(document).ready(function() {
         document.l10n.formatValue('verifyingFile').then(verifyingFile => {
           $('.progress-text').text(verifyingFile);
         });
-      } else {
-        console.log('Finished hashing');
       }
     });
 
@@ -219,7 +217,6 @@ $(document).ready(function() {
           $('.progress-text').text(encryptingFile);
         });
       } else {
-        console.log('Finished encrypting');
         uploadStart = Date.now();
       }
     });

--- a/server/server.js
+++ b/server/server.js
@@ -33,12 +33,7 @@ function allLangs() {
 }
 
 function prodLangs() {
-  // eslint-disable-next-line security/detect-non-literal-require
-  return require(path.join(
-    __dirname,
-    '..',
-    'package.json'
-  )).availableLanguages.join(',');
+  return require('../package.json').availableLanguages.join(',');
 }
 
 const availableLanguages = conf.l10n_dev ? allLangs() : prodLangs();

--- a/server/server.js
+++ b/server/server.js
@@ -33,6 +33,7 @@ function allLangs() {
 }
 
 function prodLangs() {
+  // eslint-disable-next-line security/detect-non-literal-require
   return require(path.join(
     __dirname,
     '..',

